Vagrantfile

```diff
@@ -42,7 +42,7 @@ Vagrant.configure("2") do |config|
   echo "Compiling and installing the extension..."
   pushd /usr/local/src/php-driver/ext
   phpize
-  LDFLAGS="-L$builddir/lib" LIBS="-lssl -lz -luv -lgmp -lstdc++" ./configure --with-cassandra=/tmp/php-driver-installation/build --with-gmp=/tmp/php-driver-installation/build --with-libdir=lib
+  LDFLAGS="-L$builddir/lib" LIBS="-lssl -lz -luv -lm -lgmp -lstdc++" ./configure --with-cassandra=/tmp/php-driver-installation/build --with-gmp=/tmp/php-driver-installation/build --with-libdir=lib
```

Comment:
Logarithmic functions are being used (https://github.com/datastax/php-driver/blob/master/ext/util/math.c#L427) and were not being linked in (e.g. undefined references)